_includes/volunteer/main.html

```diff
@@ -0,0 +1,297 @@
+<div class="jumbotron background-white">
+<form id="volunteer-form" name="volunteer" enctype="multipart/form-data" v-on:submit.prevent="addVolunteer"> 
+  <div class = "row">
+    <div class="col-md-7 col-lg-5">
+      <div class="form-group">
+        <input type="text" id="firstName"  v-model="newVolunteer.firstName" required class="form-control volunteerField">
+        <label for="firstName" :class="{'stateField': newVolunteer.firstName}" class="form-control-placeholder">First Name</label>
```

Comment:
The `for` attribute on the `label` works with the `name` attribute on `input`. We'll have to add the `name` attributes. :(